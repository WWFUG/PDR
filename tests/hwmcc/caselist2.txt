6s6.aig
6s136.aig
6s206rb025.aig
6s221rb18.aig
6s307rb06.aig
6s326rb02.aig
6s327rb10.aig
6s374b029.aig
6s380b129.aig
6s388b07.aig
abp4pold.aig
bob9234spec7neg.aig
bobpci215.aig
pdtpmsfpmult.aig
pj2018.aig

