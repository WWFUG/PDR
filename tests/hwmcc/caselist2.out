v3> read aig 6s6.aig

v3> satv pdr -o 0

TIMEOUT
v3> read aig 6s136.aig

v3> satv pdr -o 0
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
